response/response.go

```diff
@@ -32,6 +32,12 @@ import (
 // DefaultTTL is the default TTL for which a response can be cached.
 const DefaultTTL = 1 * time.Minute
 
+// ResultOption allows you to further configure the Result that gets added to
+// the response.
+type ResultOption struct {
```

Comment:
```suggestion
// ResultOption allows you to further configure the Result that gets added to
// the response.
type Result struct {
```

Would just `Result` make sense here?

Reply:
updated to `Result` and changed the description of the struct to match Result in pb